Makefile

```diff
@@ -631,6 +636,16 @@ else
 endif
 	@echo "makefile: Done running tests"
 
+.PHONY: installtest
+installtest: $(bdist_file) $(sdist_file) tests/install/test_install.sh
+	@echo "makefile: Running install tests"
+ifeq ($(PLATFORM),Windows_native)
+	@echo "makefile: Warning: Skipping install test on native Windows" >&2
```

Comment:
Lets start an issue for no install test on windows.

Reply:
I approved the pr since this is not really part of the pr itself

Reply:
DONE - created issue #607.